### DISCUSSION TITLE
Unable to open RavenDB Setup Wizard from Powershell Core

### DISCUSSION THREAD

**Discussion by romanov** [ravendb/ravendb#16705] [Q&A]
I've installed RavenDB on my remote Ubuntu server.
I've opened 53700 port. But unable to open it in my browser.

Looks like the port 53700 is blocked by RavenDB itself & i am unable to port forward from windows to ubuntu.

Is it possible to get to the Setup Wizard ?

**Comment by ayende** [comment]
If you are on a remote server, you need to setup port tunneling to that location.
Make sure to include port `53700` (note, it is usually dynamic) and then run the setup
